validation-test/Sema/SwiftUI/rdar88256059.swift

```diff
@@ -9,7 +9,8 @@ struct MyView: View {
 
   var body: some View {
     Table(self.data) {
-      // expected-error@-1 {{expected expression of type 'Columns' in result builder 'TableColumnBuilder'}} {{23-23=<#T##Columns#>}}
+      // expected-error@-1 {{expected expression of type 'Column' in result builder 'TableColumnBuilder'}} {{23-23=<#T##Column#>}}
+      // expected-error@-2 {{generic parameter 'Column' could not be inferred}}
```

Comment:
This is a little unfortunate, but it's because we're now no longer stripping `TVO_CanBindToHole` from the generic parameter. We can't eagerly bind the missing argument to a hole because we rely on type inference to infer the type for better diagnostics. Maybe we could rework those diagnostics to instead look at the parameter type, or maybe we could tweak the diagnostic logic for a defaulted generic parameter to account for missing argument fixes (unfortunately it currently gets defaulted before we match arguments though). In any case, I'm going to leave this as future work.